testing/ef_tests/src/cases.rs

```diff
@@ -89,26 +89,26 @@ pub use transition::TransitionTest;
 ///     to return `true` for the feature in order for the feature test vector to be tested.
 #[derive(Debug, PartialEq, Clone, Copy)]
 pub enum FeatureName {
-    Eip7594,
+    Placeholder,
```

Comment:
Keeping this around as it's likely going to be useful for feature prototyping in the future.